R/utils.R

```diff
@@ -277,6 +277,10 @@ stat_hypotheses <- tibble::tribble(
   "sd",                "point",
   "prop",              "point",
   "count",             "point",
+  "mean",              "paired independence",
+  "median",            "paired independence",
+  "sum",               "paired independence",
+  "sd",                "paired independence",
```

Comment:
Surfaced each of the options that seem plausible, though not sure if we want to limit this to fewer.

Reply:
Anything we're allowing with non-paired independence should be fair game here as well.